playwright/e2e/crypto/device-verification.spec.ts

```diff
@@ -124,6 +124,10 @@ test.describe("Device verification", { tag: "@no-webkit" }, () => {
         const toasts = new Toasts(page);
         await toasts.rejectToast("Notifications");
         await toasts.assertNoToasts();
+
+        // There may still be a `/sendToDevice/m.secret.request` in flight, which will later throw an error and cause
+        // a *subsequent* test to fail. Tell playwright to ignore any errors resulting from in-flight routes.
+        await page.unrouteAll({ behavior: "ignoreErrors" });
```

Comment:
for clarity: this is undoing the `page.route` at line 101, but more importantly, the `ignoreErrors` tells playwright that if any such routes are still in flight, it should ignore any exceptions that get thrown by it.

(The problem is that `route.fetch()` will fail if it runs after the test has finished)